index.html

```diff
@@ -32,9 +39,26 @@ <h2>Projects</h2>
     </section>
 
     <section class="work-section">
+      <div class="projectcombine">
+        <div class="proRotate">
+          <h1>Projects</h1>
+          <img src="/Asset/dots2.svg" class="dots" alt="hi" />
+        </div>
+        <div class="backimg1"></div>
+      </div>
       <div class="grid-container">
        
-          
+        <div class="grid-it">
+          <h2>Project name goes here</h2>
+          <ul>
+            <li class="html">HTML/CSS</li>
+            <li class="ruby">Ruby on Rails</li>
+            <li class="javas">Javascript</li>
+          </ul>
+          <button class="buttn">See this project</button>
```

Comment:
- [ ] Kindly fix the font center the text to match the Figma design,

-Your design:
![Screenshot from 2023-03-04 19-04-04](https://user-images.githubusercontent.com/97834160/222919145-8f51bf1d-c623-47ea-8e3a-a85c4dd9021c.png)
![Screenshot from 2023-03-04 19-03-59](https://user-images.githubusercontent.com/97834160/222919152-c2977c5c-76c6-426a-843f-5df61490a1df.png)


- Figma:
![Screenshot from 2023-03-04 18-58-12](https://user-images.githubusercontent.com/97834160/222919051-6e543cbe-daf8-4788-b65a-6a3a2f1ab3a9.png)